Moonshot/items_objects/FireRatePickup.gd

```diff
@@ -0,0 +1,33 @@
+extends RigidBody2D
+
+# Declare member variables here. Examples:
+# var a = 2
+# var b = "text"
+
+var shot_speed = 0.01
```

Comment:
is this the speed of the shot or the speed of the pickup?

it seems to be the speed of which the pickup travels toward the player, maybe `const PICK_UP_TRAVEL_SPEED`

Reply:
Yep will change this too